cookbook/src/tx_segwit-v0.md

```diff
@@ -0,0 +1,351 @@
+# Constructing and Signing Transactions - SegWit V0
+
+In this section, we will construct a [SegWit V0 transaction](https://bitcoinops.org/en/topics/segregated-witness/).
+This is the most common type of transaction on the Bitcoin network today[^today].
+
+This is the `cargo` commands that you need to run this example:
+
+```bash
+cargo add bitcoin --features "std, rand-std"
```

Comment:
@tcharding, I think this is bug in rust-bitcoin. I would expect `rand-std` to include `std`.

Reply:
https://github.com/rust-bitcoin/rust-bitcoin/pull/2102